### PR TITLE
Complete HW #5

### PR DESCRIPTION
**Pull request by AnastasiiaNashchokina**

### CODE REVIEW

**Comment by AntBelyi** [comment]
I don't see all your merged tasks
image

File: src/main/java/HW_5/myCity/Building.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package HW_5.myCity;
+
+import HW_5.myClass.Person;
+
+public class Building {
+    public Person[] persones;
```
Comment (AntBelyi) [inline src/main/java/HW_5/myCity/Building.java]: 1) mistake in "persones" word
2) Please add some information to Person class(name, surname, age, toString()), create few persons in main method in Building class, add these persons to array and using cycle display persons from array

File: src/main/java/HW_5/Main.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        Point[] points = new Point[4];
+        points[0] = new Point(2, 2);
+        points[1] = new Point(3, 7);
+        points[2] = new Point(4, 6);
+        points[3] = new Point(5, 7);
+        System.out.println();
+        for (Point point : points) {
+            point.info();
+        }
+        System.out.println();
```
Comment (AntBelyi) [inline src/main/java/HW_5/Main.java]: What is this line doing?

File: src/main/java/HW_5/myCity/Building.java (hunk 1 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_5/myCity/Building.java]: fixed

File: src/main/java/HW_5/Main.java (hunk 2 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_5/Main.java]: fixed

Comment (AntBelyi) [reply src/main/java/HW_5/Main.java]: not

File: src/main/java/HW_5/Main.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package HW_5;
+
+public class Main {
+    public static void main(String[] args) {
+        Point[] points = new Point[4];
+        points[0] = new Point(2, 2);
+        points[1] = new Point(3, 7);
+        points[2] = new Point(4, 6);
+        points[3] = new Point(5, 7);
+        System.out.println();
```
Comment (AntBelyi) [inline src/main/java/HW_5/Main.java]: What is this line doing?

File: src/main/java/HW_5/myClass/Person.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        this.age = age;
+    }
+
+    @Override
+    public String toString() {
+        return "Person{" +
+                "name='" + name + '\'' +
+                ", surname='" + surname + '\'' +
+                ", age=" + age +
+                '}';
```
Comment (AntBelyi) [inline src/main/java/HW_5/myClass/Person.java:33]: Please remove unused methods in Person class

File: src/main/java/HW_5/Main.java (hunk 2 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_5/Main.java]: fixed

File: src/main/java/HW_5/Main.java (hunk 3 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_5/Main.java]: fixed

File: src/main/java/HW_5/myClass/Person.java (hunk 4 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/HW_5/myClass/Person.java:33]: fixed

**Pull request merged**